### PR TITLE
sequencer should connect to celestia-da (26650 port)

### PR DESCRIPTION
**Pull request by okdas**
Switches sequencer to connect to `26650` port of `local-celestia-devnet` container.

Related changes: https://github.com/pokt-network/poktroll/pull/367

### CODE REVIEW

File: charts/poktroll-sequencer/templates/deployment.yaml [hunk 1]
```diff
@@ -57,12 +57,12 @@ spec:
...
           securityContext:
 {{ toYaml .Values.podSecurityContext | indent 12 }}
-          env:
-            - name: AUTH_TOKEN
-              valueFrom:
-                secretKeyRef:
-                  name: {{ .Values.dataAvailability.authToken.secretName }}
-                  key: {{ .Values.dataAvailability.authToken.secretKey }}
+          # env:
+          #   - name: AUTH_TOKEN
```
Comment (Olshansk) [inline charts/poktroll-sequencer/templates/deployment.yaml:61]: Can we delete this altogether?

File: charts/poktroll-sequencer/templates/deployment.yaml [hunk 2]
```diff
@@ -81,9 +81,7 @@ spec:
           command: ["/bin/sh", "-c"]
           args: 
             - |
```
Comment (Olshansk) [inline charts/poktroll-sequencer/templates/deployment.yaml:83]: Don't we have to specify the block time via a flag as well?

File: charts/poktroll-sequencer/templates/deployment.yaml [hunk 3]
```diff
@@ -81,9 +81,7 @@ spec:
...
-              export DA_BLOCK_HEIGHT=$(cat /root/.pocket/data/da-block-height)
-              export CELESTIA_NAMESPACE=$NAMESPACE
-              exec  poktrolld start --home=/root/.pocket/ --rollkit.aggregator=true --rollkit.da_address="{{ .Values.dataAvailability.celestia.da_rpc_url }}" --rollkit.da_start_height=$DA_BLOCK_HEIGHT --rpc.laddr=tcp://0.0.0.0:36657 --p2p.laddr="0.0.0.0:36656" --grpc.address=0.0.0.0:36658
+              exec poktrolld start --home=/root/.pocket/ --rollkit.aggregator=true --rollkit.da_address="{{ .Values.dataAvailability.celestia.da_rpc_url }}" --rollkit.da_start_height=1 --rpc.laddr=tcp://0.0.0.0:36657 --p2p.laddr="0.0.0.0:36656" --grpc.address=0.0.0.0:36658
```
Comment (Olshansk) [inline charts/poktroll-sequencer/templates/deployment.yaml:84]: Why is the start height 1?

**Review by Olshansk — CHANGES_REQUESTED** [review]

File: charts/poktroll-sequencer/templates/deployment.yaml (hunk 3 above)
Comment (okdas) [reply charts/poktroll-sequencer/templates/deployment.yaml:84]: @Olshansk I just wanted to remove the unnecessary complexity for the chart we use for development. There isn't really any difference for local development.

File: charts/poktroll-sequencer/templates/deployment.yaml (hunk 2 above)
Comment (okdas) [reply charts/poktroll-sequencer/templates/deployment.yaml:83]: @Olshansk we can, but we don't have to. This chart is for local development, so default block times work for us.

**Pull request merged**